anesthetic/tension.py

```diff
@@ -0,0 +1,105 @@
+"""Tension statistics between two datasets."""
+from anesthetic.samples import Samples
+from scipy.stats import chi2
+import numpy as np
+
+
+def stats(A, B, AB, nsamples=None, beta=None):  # noqa: D301
```

Comment:
@lukashergt @DilyOng for multiple datasets, I think unpacking will neatly handle arbitrary numbers of datasets, something like

```python
def stats(h0, *h1, nsamples=None, beta=None):
    ```h0 = null hypothesis = AB, h1stats = alternative hypothesis = A, B etc```
    ...
    samples['logR'] = h0stats['logZ'] - sum(_h1stats['logZ'] for _h1stats in h1stats)
    ...
```

which can be called `tension.stats(abcde, a, b, c, d, e)`

Reply:
Yes, that's about what I have. I call them `joint` and `separate`, which I find a bit more descriptive...